frontend/app/controllers/user.js

```diff
@@ -2,22 +2,23 @@ import Ember from 'ember';
 import HasCoverUpload from '../mixins/has-cover-upload';
 import ajax from 'ic-ajax';
 
-export default Ember.ObjectController.extend(HasCoverUpload, {
+export default Ember.Controller.extend(HasCoverUpload, {
   showEditMenu: false,
 
-  viewingSelf: function () {
+  viewingSelf: function() {
     return this.get('model.id') === this.get('currentUser.id');
   }.property('model.id'),
 
-  forumProfile: function () {
+  forumProfile: function() {
     return "https://forums.hummingbird.me/users/" + this.get('model.username');
   }.property('model.username'),
 
   // Legacy URLs
-  feedURL: function () {
```

Comment:
I think I removed both of these routes from `routes.rb` so these should probably get nuked too